custom/icds_reports/views.py

```diff
@@ -755,6 +755,7 @@ def post(self, request, *args, **kwargs):
         if indicator == BENEFICIARY_LIST_EXPORT:
             if not sql_location or sql_location.location_type_name in [LocationTypes.STATE]:
                 return HttpResponseBadRequest()
+            config = beneficiary_config
```

Comment:
where does `beneficiary_config` come from

Reply:
Hi,
It comes from lines 708 to 733, it had to be missed by me when I changed the download of those reports to be made asynchronously:
```        beneficiary_config = {'domain': self.kwargs['domain'], 'filters': request.POST.getlist('filter[]')}


        if month and year:
            beneficiary_config['month'] = date(year, month, 1)
            config.update({
                'month': date(year, month, 1),
            })


        location = request.POST.get('location', '')


        sql_location = None


        if location and indicator != ISSNIP_MONTHLY_REGISTER_PDF:
            try:
                sql_location = SQLLocation.objects.get(location_id=location, domain=self.kwargs['domain'])
                locations = sql_location.get_ancestors(include_self=True)
                for loc in locations:
                    location_key = '%s_id' % loc.location_type.code
                    config.update({
                        location_key: loc.location_id,
                    })
                    beneficiary_config.update({
                        location_key: loc.location_id
                    })
            except SQLLocation.DoesNotExist:
                pass```